webapp/src/migrations/1.9.0.ts

```diff
@@ -0,0 +1,14 @@
+import debug from 'debug';
+const log = debug('webapp:migration:1.9.0');
+
+export default async function (db) {
+	log('renaming chatAppConfig.recursionLimit to maxMessages');
+	await db.collection('apps').updateMany(
+		{},
+		{
+			$rename: {
+				'chatAppConfig.recursionLimit': 'chatAppConfig.maxMessages'
```

Comment:
@charl3sj can you update package.json version to 1.9.0 to correspond with this? Good job on already making the migration :smile: